packages/smooth_app/lib/pages/folksonomy/folksonomy_create_edit_modal.dart

```diff
@@ -129,38 +134,46 @@ class _FolksonomyEditTagContentBody extends StatelessWidget {
             explanation: appLocalizations.tag_key_explanations,
             hasErrors: !isKeyValid,
           ),
-          TextField(
+          SimpleInputTextField(
```

Comment:
It means that we cannot input "new" keys or values, right?
Is that what was wanted initially?

Reply:
no, we want to be able to add new ones. We even added a specific pipeline on the web that allows creating a documentation page on the wiki (we can do that in a future PR)

Reply:
@teolemon Actually I've just tested on the app with "category": when you type in a text there's a search on the server for existing values but it is possible to add new values regardless.